src/Components/Facility/TriageForm.tsx

```diff
@@ -263,54 +251,29 @@ export const TriageForm = (props: triageFormProps) => {
           },
         }}
         backUrl={`/facility/${facilityId}`}
-      />
-
-      <Modal
-        open={openModalForExistingTriage}
-        aria-labelledby="Triage Check"
-        aria-describedby=""
-        className=""
       >
-        <div className="h-screen w-full absolute flex items-center justify-center bg-modal">
-          <div className="bg-white rounded shadow p-8 m-4 max-h-full text-center flex flex-col max-w-lg w-2/3 min-w-max-content">
-            <div className="mb-4">
-              <i className="fa-solid fa-triangle-exclamation text-red-500 fa-4x"></i>
-              <h1 className="sm:text-xl text-sm">
+        <ConfirmDialogV2
+          title={
+            <div className="">
+              <i className="fa-solid fa-triangle-exclamation text-red-500 fa-3x"></i>
+              <h1 className="sm:text-xl text-sm mt-4">
                 A Triage already exist on this date
               </h1>
-              <p className="text-base">
-                If you wish to proceed then the existing triage will be over
-                written!
-              </p>
             </div>
-            <div></div>
-            <div className="flex flex-col-reverse md:flex-row gap-2 mt-4 justify-end">
-              <ButtonV2
-                variant="secondary"
-                onClick={() => {
-                  setOpenModalForExistingTriage(false);
-                }}
-              >
-                Cancel
-              </ButtonV2>
-              <ButtonV2
-                variant="danger"
-                id="triageConfirm"
-                onClick={(e) => {
-                  handleSubmit(e);
-                }}
-              >
-                Proceed
-              </ButtonV2>
-            </div>
-          </div>
-        </div>
-      </Modal>
+          }
+          description="A Triage already exist on this date,  If you wish to proceed then the existing triage will be over
+          written!"
+          variant="danger"
+          show={openModalForExistingTriage}
+          onClose={() => setOpenModalForExistingTriage(false)}
+          className="w-[48rem]"
+          action={<div id="triageConfirm">Proceed</div>}
+          onConfirm={() => handleSubmit}
+        />
 
-      <div className="mt-4">
-        <Card>
-          <form onSubmit={(e) => handleSubmit(e)}>
-            <CardContent>
+        <div className="mt-4">
+          <Card>
+            <form onSubmit={() => handleSubmit}>
```

Comment:
You are not calling this???
```suggestion
            <form onSubmit={handleSubmit}>
```